lib/ui/code_widget.dart

```diff
@@ -383,6 +388,10 @@ class _CodeWidgetState extends State<CodeWidget> {
   }
 
   Future<void> _onShowQrPressed(_) async {
+    bool _isAuthSuccessful = await requestAuthentication("Authorize to shoq QR");
```

Comment:
Ditto

Reply:
I added the string to all the arb files except app_he.arb since he locale is not getting used.